integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioManagedCoherence.java

```diff
@@ -156,7 +164,7 @@ public static void init(@Namespaces(2) List<String> namespaces) {
    */
   @Test
   @DisplayName("Two cluster domain with a Coherence cluster with ISTIO and test interaction with cache data")
```

Comment:
@DisplayName("Two cluster domain-in-image domain with a Coherence cluster with ISTIO and test interaction with cache data")